metagraph/src/common/sorted_sets/sorted_set_disk_base.cpp

```diff
@@ -146,6 +146,19 @@ void SortedSetDiskBase<T>::dump_to_file(bool is_done) {
     chunk_count_++;
 }
 
+template <typename T>
+void SortedSetDiskBase<T>::insert_sorted(const std::vector<T> &data) {
+    std::string file_name = chunk_file_prefix_ + "sorted";
+    constexpr bool append = true;
+
+    EliasFanoEncoder<T> encoder(data.size(), utils::get_first(data.front()),
+                                utils::get_first(data.back()), file_name, append);
+    for (const auto &v : data) {
+        encoder.add(v);
+    }
```

Comment:
what about `get_second`?

Reply:
Ah, all good!
I was confused, thought you pass only iterators to utils::get_first elements.